### PR TITLE
COG-274 - Adding container security scan stage

### PR DESCRIPTION
**Pull request by ratneshraval**
### JIRA Issue Link
- https://osi-cwds.atlassian.net/browse/COG-274

### Technical Description
- run tenable scan on perry container

### Tests
- [ ] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

### Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Configuration changes
N/A

### Technical debt
N/A

### Checklist:

- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by ratneshraval** [comment]
Jenkins retest this please

**Comment by ratneshraval** [comment]
jenkins retest this please

File: Jenkinsfile-pull-request.groovy [hunk 1]
```diff
@@ -24,10 +24,16 @@ node('dora-slave') {
             if (params.RELEASE_PROJECT) {
                 echo "!!!! BUILD RELEASE VERSION"
                 buildInfo = rtGradle.run buildFile: 'build.gradle', tasks: 'clean jar -DRelease=$RELEASE_PROJECT -DBuildNumber=$BUILD_NUMBER -DCustomVersion=$OVERRIDE_VERSION'
+                echo "--------------------------------------"
```
[the remark this answers is not in the document]
Comment (ratneshraval) [reply Jenkinsfile-pull-request.groovy]: I added those for testing but I'll remove that. I changed PR title to WIP for now until it is ready for review.

**Comment by sorych** [comment]
jenkins retest this please

**Comment by ratneshraval** [comment]
jenkins retest this please

**Comment by ratneshraval** [comment]
jenkins retest this please

**Comment by ratneshraval** [comment]
jenkins retest this please

**Comment by ratneshraval** [comment]
Version number is available now in Jenkinsfile. Can be verified by viewing http://jenkins.dev.cwds.io:8080/view/perry/job/perry-build-pull-request/415/consoleFull
and looking for text "echo build_version:"

**Comment by ratneshraval** [comment]
Can someone merge this please? Says I can't merge because I'm not authorized user.

**Pull request merged**